cmd/shipperctl/cmd/apply.go

```diff
@@ -120,6 +128,18 @@ func setupManagementCluster(managementCluster *config.ClusterConfiguration, cmd
 		return err
 	}
 
+	if err := createValidatingWebhookSecret(cmd, configurator); err != nil {
+		return err
```

Comment:
Do you think it might make sense to decorate the errors (this and 2 more below)? like: ```return fmt.Errorf("failed to create validating webhook secret: %s", err)```

Reply:
Do you mean that we should do this for more visibility on the user's side? In that case, I think it's already being done.

An example of output from `shipperctl` first:

```
Setting up management cluster microk8s:
Registering or updating custom resource definitions... done
Creating a namespace called shipper-system... done
(...snipped...)
Creating a secret for the validating webhook:
    Generating a private key... done
    Creating a TLS certificate signing request... done
    Creating a Kubernetes CertificateSigningRequest... done
    Approving the CertificateSigningRequest... error! unable to create new content in namespace shipper-system because it is being terminated
```

Does this do what you're suggesting?

Reply:
It does, yes. I was mostly concerned about straight error forwarding which sometimes looses the context of it's occurrence. Doesn't seem to the case here.